src/go/rpk/pkg/cli/cmd/cluster/config/import.go

```diff
@@ -100,6 +106,12 @@ func importConfig(
 
 	for k := range oldConfig {
 		if _, found := readbackConfig[k]; !found {
+			// We are excluding cluster_id since is a variable stored as a config
+			// option and we don't want to replicate from one cluster to another.
```

Comment:
I'd drop these two doc lines because they duplicate they above, and change `if k == .. {` to add `// see above` at the end.